modules/gapi/test/common/gapi_tests_helpers.hpp

```diff
@@ -50,6 +50,18 @@ namespace opencv_test
     __TUPLE_PARAM_TYPE(index) param_name = getSpecificParam<index>(); \
     __WRAP_VAARGS(__DEFINE_PARAMS_IMPL7(index+1, __VA_ARGS__))
 
+#define __DEFINE_PARAMS_IMPL9(index, param_name, ...) \
+    __TUPLE_PARAM_TYPE(index) param_name = getSpecificParam<index>(); \
+    __WRAP_VAARGS(__DEFINE_PARAMS_IMPL8(index+1, __VA_ARGS__))
```

Comment:
Looks like this is not related to the patch

Reply:
That was rebase mistake, fixed now.